### PR TITLE
HL-23

### PR DESCRIPTION
**Pull request by brettvniekerk**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-23)

- Add history table of all vault transactions to vaults page

image

### CODE REVIEW

File: src/providers/Apollo.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {
+  ApolloClient,
+  HttpLink,
+  InMemoryCache,
+  ApolloProvider
+} from "@apollo/client";
+import React from "react";
+
+const uri =
+  "https://api.thegraph.com/subgraphs/name/horse-link/hl-protocol-goerli";
```
Comment (xiangzhe0812) [inline src/providers/Apollo.tsx:10]: lets put url into .env

**Review by C00p3rb123 — CHANGES_REQUESTED** [review]
Round the decimal places please

**Comment by brettvniekerk** [comment]
> Round the decimal places please

This is will probably be resolved as part of [this ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82/backlog?selectedIssue=HL-128)

**Review by pondpiu — DISMISSED** [review]

**Review by C00p3rb123 — DISMISSED** [review]

File: src/providers/Apollo.tsx (hunk 1 above)
Comment (brettvniekerk) [reply src/providers/Apollo.tsx]: 👍

**Review by pondpiu — APPROVED** [review]

**Review by C00p3rb123 — APPROVED** [review]

**Pull request merged**